tests/e2e/pageobjects/openshift/CheLoginPage.ts

```diff
@@ -41,7 +42,7 @@ export class CheLoginPage {
     async clickEclipseCheLoginButton() {
         Logger.debug('CheLoginPage.clickEclipseCheLoginButton');
 
-        await this.driverHelper.waitAndClick(By.id('kc-login'));
+        await this.driverHelper.waitAndClick(By.id('kc-login'), TimeoutConstants.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM);
```

Comment:
Passing this variable also seems unecessary. 
I think this is pretty common in this PR, so I'll stop commenting on all of the occurences of this.

Reply:
Fixed, I tried to fix also all other occurences.